src/screens/reader/components/ReaderBottomSheet/ReaderValueChange.tsx

```diff
@@ -3,43 +3,61 @@ import React from 'react';
 
 import { useChapterReaderSettings, useTheme } from '@hooks/persisted';
 import { IconButtonV2 } from '@components';
-import { getString } from '@strings/translations';
 
-interface ReaderLineHeightProps {
+type ValueKey<T extends object> = {
+  [K in keyof T]: T[K] extends number ? K : never;
+}[keyof T];
+
+interface ReaderValueChangeProps {
   labelStyle?: TextStyle | TextStyle[];
+  valueChange?: number;
+  label: string;
+  valueKey: ValueKey<ReturnType<typeof useChapterReaderSettings>>;
```

Comment:
```suggestion
  valueKey: ValueKey<ChapterReaderSettings>;
```
for less scary, remember to import `ChapterReaderSettings`

Reply:
Makes sense, I only thought of checking for a type with the hook declaration.